dmc/configs/visualizations.py

```diff
@@ -0,0 +1,49 @@
+# Copyright 2025 Thousand Brains Project
+# Copyright 2023 Numenta Inc.
```

Comment:
If you just created this file in 2025, you don't need to include Numenta in the copy right header. If you created it last year, it should say 2024 (https://thousandbrainsproject.readme.io/docs/style-guide#source-code-copyright-and-license-header)